__tests__/utility.service.test.js

```diff
@@ -19,18 +31,21 @@ describe("UtilityService", () => {
   });
 
   describe("updateSearchResults", () => {
-    it("should call provided functions", () => {
+    it("should call provided functions and ", () => {
```

Comment:
Would prefer to have a more descriptive test name :P

Reply:
Yeah its currently incomplete